modin/data_management/functions/default_methods/resample_default.py

```diff
@@ -15,8 +15,27 @@
 
 
 class Resampler:
+    """Builder class for resampled aggregation functions."""
+
     @classmethod
     def build_resample(cls, func, squeeze_self):
+        """
+        Build function that resamples time-series data and does aggregation.
+
+        Parameters
+        ----------
+        func : callable
+            Aggregation function to execute under resampled frame.
+        squeeze_self : bool
+            Whether or not to squeeze frame before resampling.
```

Comment:
I don't think we need to document this notion because it is ordinary pandas operation aka `squeeze` on our underlying object. So, let's leave as it is.